src/tools/cephfs/JournalTool.cc

```diff
@@ -728,6 +747,12 @@ int JournalTool::recover_dentries(
 
   int r = 0;
 
+  // Initialize progress tracking for dentry recovery
+  if (metablob.lump_order.size() > 1) {
```

Comment:
I think progress tracker should be initialized even when there is a single entry in lump_order.